docs/upgrade/upgrade-7.17-8.x.asciidoc

```diff
@@ -0,0 +1,155 @@
+
+* Choose your desired {elastic-sec} 8.x version to upgrade to. We recommend upgrading to the latest minor and patch version, as this provides the most extensive set of features, bug fixes, and security patches. Be sure to use the same version for upgrading your development or non-production deployment as for your production deployment. 
+
+* Review the latest 8.x {elastic-sec} features and the current set of Elastic connectors, integrations, and detection rules to determine if you can replace any customized content with out-of-the-box functionality. This can help reduce the workload and complexity of your upgrade.
+
+* If your organization uses external SOAR (security orchestration, automation, and response) workflows on {elastic-sec} alerts (signals), you may need to change your workflows to accommodate the new <<alert-schema, alert schema>> used in 8.x. These changes may take up to two weeks to complete.
+
+* Review release notes, deprecations, and breaking changes for {elastic-sec}, {es}, {kib}, and, if applicable, {fleet}, {agent}, {beats}, and {ls}. Identify any issues that might affect your deployment. Work with your Support Engineer on any questions you may have. Start with breaking changes for solution and platform components you use, such as {es} and {kib}. 
+
+* Schedule a system maintenance window within your organization.
+
+[float]
+=== Pre-upgrade steps
+
+To prepare for the upgrade process, follow these steps before you start:
+
+. Do a software version inventory across your entire Elastic platform, including {es}, {kib}, {agent}, {beats}, and {ls}. If you're managing your deployments using {ece-ref}/ece-upgrade.html[{ece}] (ECE) or {eck-ref}/k8s-upgrading-eck.html[{eck}] (ECK), you may need to upgrade the orchestrator first.
+
+. If you're running any of these at versions earlier than 7.17, you must first upgrade them all to the latest 7.17.x patch release before upgrading to 8.x. This enables you to use the {kib} Upgrade Assistant to upgrade to 8.x. 
+
+. Run the {kib-ref}/upgrade-assistant.html[Upgrade Assistant]. Take note of any critical error messages, then work with your Support rep to resolve them.
++
+.Requirements
+[sidebar]
+--
+To run the Upgrade Assistant, you must have `superuser` cluster admin permissions.
+--
+
+. If you're not using {ref}/snapshots-take-snapshot.html#automate-snapshots-slm[{slm} (SLM)], you must set up, configure, and run at least one {ref}/snapshot-restore.html[snapshot]. Snapshots are {ref}/snapshot-restore.html#how-snapshots-work[incremental] – depending on the cluster size and the input/output rate, the initial snapshot may take several hours to complete. If you are using SLM, {ref}/snapshots-take-snapshot.html#check-slm-history[check the SLM history] to ensure that snapshots are completing successfully.
+
+[float]
+=== Perform 8.x upgrade on Non-Prod (Optional) or Prod Deployment
```

Comment:
This is good, but do we need a similar at the end like:  "Repeat this process for your Prod deployment" (If you did the above for your non-prod/dev deployments)?

Also, where do we put the comment about "If you're using a Cross-cluster search environment, make sure to upgrade your central/remote??? deployments first."  ??  Then add a "Repeat this process for other deployments, etc.

Reply:
Hi @MikePaquette, added these [additions](https://security-docs_bk_5186.docs-preview.app.elstc.co/guide/en/security/master/upgrade-7.17-8x.html#_perform_8_x_upgrade_on_a_non_prod_optional_or_prod_deployment): 
![2024-05-23_11-40-44](https://github.com/elastic/security-docs/assets/57149392/ca0bcec2-e0dc-40a8-b9c9-b244689b2a07)

and towards the end of the non-prod instructions. 
![2024-05-23_11-41-59](https://github.com/elastic/security-docs/assets/57149392/3fcebbc1-0ded-4961-8f99-3565bd430bf2)

Please let me know if these are correct, thanks!